app/assets/stylesheets/scaffolds.scss

```diff
@@ -0,0 +1,84 @@
+body {
+    background-color: #fff;
+    color: #333;
+    margin: 33px;
+}
+
+body,
+p,
+ol,
+ul,
+td {
+    font-family: verdana, arial, helvetica, sans-serif;
+    font-size: 13px;
+    line-height: 18px;
+}
+
+pre {
+    background-color: #eee;
+    padding: 10px;
+    font-size: 11px;
+}
+
+a {
+    color: white;
+    text-decoration: none;
+}
+
+a:visited {
+    color: #666;
+}
+
+a:hover {
+    text-decoration: whitesmoke;
+}
+
+th {
+    padding-bottom: 5px;
+}
+
+td {
+    padding: 0 5px 7px;
+}
+
+div.field,
+div.actions {
+    margin-bottom: 10px;
+}
+
+#notice {
+    color: green;
+}
+
+.field_with_errors {
+    padding: 2px;
+    background-color: red;
+    display: table;
+}
+
+#error_explanation {
+    width: 450px;
+    border: 2px solid red;
+    padding: 7px 7px 0;
+    margin-bottom: 20px;
+    background-color: #f0f0f0;
+}
+
+#error_explanation h2 {
```

Comment:
Heads up: The use of ID selectors in CSS should be a kind of last-option. Prefer CSS classes.

This is important when your project begins to grow, there are priorities in CSS selectors:
High: `!important` <- Try not to use it at all.
           inline styles <- Like `<a style="background-color: #000"></a>`
           html tags <- Using the same HTML name in CSS e.g. `div { background-color: black }`. Global scope
           IDs <- `#error`, they are unique to an element and the styles can't be reused. They are also hard to override
Low:   CSS classes <- `.form` Easy to reuse, to name, and to override. No potential global side-effects.